core/variant/type_info.h

```diff
@@ -367,4 +367,75 @@ ZERO_INITIALIZER_NUMBER(char32_t)
 ZERO_INITIALIZER_NUMBER(float)
 ZERO_INITIALIZER_NUMBER(double)
 
+template <typename T>
+inline constexpr bool is_variant_type_v = TypeInherits<Object, T>::value;
```

Comment:
What does the extra `v` stand for in `is_variant_type_v`?

Reply:
"Value". Not strictly necessary, but it follows the convention of taking template structures with a primary "value" field & extrapolating just that as a `constexpr`